### DISCUSSION TITLE
Getting free Pink tokens to Darwinia treasury(time sensitive for bigger reward)

### DISCUSSION THREAD

**Discussion by SasoLithops** [ringdao-community-guild/discussion#62] [Ideas]
**I am posting conversation from Discord and TG between Dustin,Laki,Jesse and me.**

**This is more for Darwinia treasury, I can repost there if necessary but since disscussed within DCDAO first let's start here.**

**All this can be found in #dcdao-itering Channel on Discord:**

Dustin posted this in former ambasador TG group. 
This would be fantastic hopefully free promotion of Darwinia. Time sensitive also. I play this game as do many(potential Pink meme token airdrop)....not easy for many to get in top 1000. Try it out. Many top parachains already connected. We can also put this up for vote but as I said time sensitive and I am sure community up for it 😀
<@1049195169850859540> <@831721232633430058> <@953495151995674674>

👇

Something darwinia might be interested in? https://twitter.com/pinkonomic/status/1743319271529156649?t=i0CRCSrEcPNonLhFPhL3-w&s=19

I would like to add to this that if the Darwinia is the 6-th parachain, 0.74% of $pink Total Supply will be for Darwinia community.
But not just this, if we could list $pink on Snowswap lol or maybe find some creative way to attract members of the Polkadot ecosystem who are not in Darwinia, to attract them to us with something related to $pink.
Not to mention that 0.74% of Total Supply could potentially be worth a lot of money and we could find a further fair way to distribute it to the community or maybe do something with that money and find an answer to the eternal question Wen Binance 😅

Of course I'm joking Wen Binance, but we can find a fair way to distribute those 0.74% Total Supply $pink tokens to the community and create hype for $ring where people would buy $ring and/or staking just to be eligible for those free $pink tokens.

More info: 
website: https://dotispink.xyz/
Pink game: https://dotispink.xyz/game.html
TG: https://t.me/PINKonomic
Twitter: https://twitter.com/pinkonomic

https://twitter.com/pinkonomic/status/1742562385318072795
https://twitter.com/pinkonomic/status/1743319271529156649

https://fxtwitter.com/alice_und_bob/status/1738198152144880020

https://apps.moonbeam.network/moonbeam/referendum/36

**Comment by SasoLithops** [comment]
And to add I think we have channel open with Moonbeam so hopefully not much work extra to be done🤞

**Comment by 2xJDubs** [comment]
Think this is a great idea.  Thanks for posting it Saso.

**Comment by hackfisher** [comment]
@AurevoirXavier Can you elaborate how to register the PINK? Maybe we need a proposal for voting, it looks good for council members and technical members? Might need to set some days for democracy voting?

**Reply by SasoLithops** [reply]
@hackfisher I can not tag Xavier. I asked in their Official TG chat:![Screenshot_2024-01-17-08-47-04-978_org.telegram.messenger.jpg](https://github.com/dcdao/discussion/assets/101176902/bb27f313-554a-4c07-9d42-f57adfff26ce)




hey Saso, you can likely share the following details of the PINK token with the Darwinia team:

Multilocation: { “parents”: 1, “interior”: {“X3”: [ { “Parachain”: 1000 }, {“PalletInstance”: 50}, { “GeneralIndex”: 23 }]}}
Decimals: 10
Name: PINK

**Reply by SasoLithops** [reply]
I also asked if any issues occur what to do. They said ask in their TG chat one will always help.

**Reply by aurexav** [reply]
I don't know too much about PINK. I'll take a look.

**Reply by SasoLithops** [reply]
Mostly all about Pink in first post on this thread😀

**Reply by SasoLithops** [reply]
![IMG_20240114_170737.jpg](https://github.com/dcdao/discussion/assets/101176902/87bffceb-1516-440a-b40e-01e4904f5537)

This is Invarch joining too. They No.7 on list.

**Comment by SasoLithops** [comment]
Not sure if this is for parachains as well, I can ask if you interested. As I wrote in first post it was time sensitive😬


PINK airdrop will be Friday 19th 11:59 PM UTC , if you haven't played the game, maybe you should try as soon as possible.    https://twitter.com/pinkonomic/status/1747728818738024694

**Comment by aurexav** [comment]
https://darwinia2.subsquare.io/council/motions/21
